### PR TITLE
enhancement(search website): Add system for de-indexing pages

### PR DESCRIPTION
**Pull request by lucperkins**
Signed-off-by: Luc Perkins <luc.perkins@datadoghq.com>

### CODE REVIEW

File: website/README.md [hunk 1]
```diff
@@ -74,6 +74,18 @@ The Algolia configuration for the site is controlled via the [`algolia.json`](./
...
 > Everything needed to configure Algolia search for vector.dev is in this repo; you should never make manual configuration changes through the Algolia dashboard.
 
+#### De-indexing pages
+
+If you need to prevent a page from being indexed, you can add `noindex: true` to the page's metadata. Here's an
+example:
+
+```yaml
+---
+title: Don't index me, bro
```
Comment (spencergilbert) [inline website/README.md:84]: lul

**Review by spencergilbert — APPROVED** [review]

**Comment by lucperkins** [comment]
Search terms like "wasm" and "webassembly" currently do *not* bring up those guides, so this seems to have done the job.

File: website/README.md (hunk 1 above)
Comment (lucperkins) [reply website/README.md:84]: A lot of memories here

**Comment by spencergilbert** [comment]
Was it also intended to not show here: https://deploy-preview-8992--vector-project.netlify.app/guides/?

**Comment by lucperkins** [comment]
@spencergilbert Nope! The goal is to keep those pages but make them less discoverable.

**Pull request merged**